microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -0,0 +1,624 @@
+
+The Microsoft Edge Add-ons API provides an API to enable your extension as sidebar extension. You can use this API to convert your extension to sidebar, and your extension will have the access to Microsoft Edge Sidebar.  You can use this *Sidebar* API by giving permissions in your `manifest.json` file.
+
+By using the Sidebar API feature for extensions, you can enhance the browsing experience by enabling users to view additional information alongside the main content of a webpage.
+
+The sidebar refers to a persistent column located on the side of the browser, which coexists with the primary content.  By enabling side-by-side browsing, users can reduce the need to constantly switch between tabs, resulting in a more productive browsing experience.
+
+
+<!-- ====================================================================== -->
+## Features of the Sidebar API
+
+* An extension in the sidebar has access to all of the [Supported APIs for Microsoft Edge extensions](./api-support.md).
+
+* The sidebar remains open while navigating between tabs.
+
+* An extension in the sidebar can be made available for specific sites.
+
+
+<!-- ------------------------------ -->
+#### Terminology
+
+| Term | Definition |
+|---|---|
+| Sidebar API | Name of the feature that you can use in your Microsoft Edge extensions. |
+| `sidepanel` | Name of the API and permission to enable any extension as a Sidebar extension, as derived from the Chromium browser engine. |
+| Sidebar extension | A Microsoft Edge extension that's designed to be opened in the sidebar. |
+
+![The sidebar for a Microsoft Edge extension](./sidebar-images/sidebar-screenshot.png)
+
+
+For details on how to create an extension, see:
+* [Create an extension tutorial, part 1](../getting-started/part1-simple-extension.md)
+* [Create an extension tutorial, part 2](../getting-started/part2-content-scripts.md)
+
+
+<!-- ====================================================================== -->
+## Origin
+
+As with other extension resources, the sidebar page commits to a trusted extension context on its origin (`extension://<id>`).  The sidebar has the same API access as other trusted extension contexts.
+
+All the existing extensions APIs are available for sidebar extensions, so you can leverage all current capabilities of the extensibility framework in your sidebar-enabled extension.
+
+To add a sidebar to your extension, follow the steps below.
+
+
+<!-- ====================================================================== -->
+## Add the sidePanel permission in the extension's manifest file
+
+<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/#manifest -->
+
+Include the `sidePanel` permission in the extension's manifest file:
+
+`manifest.json`:
+
+```json
+{
+  ...
+  "side_panel": {
+    "default_path": "sidebar.html"
+  },
+  "permissions": [
+    "sidePanel"
+  ]
+   ...
+}
+```
+<!-- todo: change to this listing?
```

Comment:
@Bhuvana-Priya - change a couple lines in this listing to be like the other page? --
https://developer.chrome.com/docs/extensions/reference/sidePanel/#manifest

Reply:
we added rather than replacing

https://github.com/MicrosoftDocs/edge-developer/pull/2648/files